cmd/keytransparency-sequencer/main.go

```diff
@@ -241,9 +224,10 @@ func runSequencer(ctx context.Context, conn *grpc.ClientConn,
 		}
 	})
 
-	sequencer.PeriodicallyRun(ctx, time.Tick(*refresh), func(ctx context.Context) {
+	go sequencer.PeriodicallyRun(ctx, time.Tick(*refresh), func(ctx context.Context) {
```

Comment:
why not use g.Go() here as well?

Reply:
At the moment these functions don't return an error.
I don't *think* we want them to because the first error causes all the other contexts decendent from `errgroup.WithContext` to exit, and we want the sequencer to carry on.